redirects.ts

```diff
@@ -316,6 +324,13 @@ export const redirectsOptions: Options = {
         if (existingPath.includes('/core')) {
             return [existingPath.replace('/core', '/oasis-core')];
         }
+        // 1041 Move Emerald and Cipher to /tools/other-paratimes/
+        if (existingPath.includes('/dapp/emerald')) {
+            return [existingPath.replace('/dapp/emerald', '/dapp/tools/other-paratimes/emerald',)];
+        }
+        if (existingPath.includes('/dapp/cipher/')) {
+            return [existingPath.replace('/dapp/cipher', '/dapp/tools/other-paratimes/cipher')];
+        }
```

Comment:
Just add `'/dapp/tools/other-paratimes/emerald'` to the existing array above.
The same for cipher.

Reply:
So just the static, no dynamic?

Reply:
Here: https://github.com/oasisprotocol/docs/blob/main/redirects.ts#L308

```
            return [
                existingPath.replace('/dapp/emerald', '/general/developer-resources/emerald-paratime'),
                existingPath.replace('/dapp/emerald', '/dapp/tools/other-paratimes/emerald')
            ];
```

Reply:
Alternatively we could ditch the `createRedirects()` entirely and list all files one by one in the `redirects` field.

Reply:
I actually don't understand the dynamic createRedirects().
Why change the existing path when it is the new correct one?

Reply:
> I actually don't understand the dynamic createRedirects(). Why change the existing path when it is the new correct one?

This function is called passing absolute URL of each file that currently exists. The function maps it to one or more historic paths which now contain redirection HTML code.